server/http_status.go

```diff
@@ -123,3 +125,24 @@ func (s *Server) handleStatus(w http.ResponseWriter, req *http.Request) {
 		terror.Log(errors.Trace(err))
 	}
 }
+
+func (s *Server) handleHTTPRoot(w http.ResponseWriter, req *http.Request) {
```

Comment:
It is better to use the [Router.Walks()](https://github.com/gorilla/mux#walking-routes) than hardcode.

Reply:
if I use Router.Walks(), I can not get /debug/pprof/ and can not get a more nice name for router, such as BinlogRecover for /binlog/recover.

Reply:
The second one is reasonable.